starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,84 @@
-// Names and Input
+//Names and Input
+
+var hacker1 = "Jules";
+var hacker2 = promt("What's your name ?"); 
+
+// console.log("The driver is " + hacker1 + ".");
+// console.log("The navigator is " + hacker2 + ".");
+
+if (hacker1.length > hacker2.length) {
+  console.log("The Driver has the longest name, it has "+ hacker1.length  + " characters.");
+} else if (hacker2.length > hacker1.length) {
+  console.log("Yo, navigator got the longest name, it has " + hacker2.length + "  characters.");
+} else {
+  console.log("wow, you both got equally long names, " + hacker1.length + " characters!!");
+}
+
+var nameUpper = hacker1.toUpperCase();
+var nameSpace = "";
+var nameReverse = "";
+
+for (var i = 0; i < nameUpper.length; i++) {
+  nameSpace += nameUpper[i] + " ";
+}
+  console.log(nameSpace);
+
+for (var i = hacker1.length - 1; i >= 0; i--) {
+  nameReverse += hacker1[i];
+}
+  console.log(nameReverse);
+
+
+if (hacker1[0] < hacker2[0]) {
```

Comment:
for this to work properly, you need to make it case insensitive. this is because you're really comparing the [character codes](https://www.w3schools.com/charsets/ref_html_ascii.asp) which is different for uppercase and lowercase letters